model-api-gen-gradle-test/src/test/kotlin/GeneratedApiTest.kt

```diff
@@ -43,6 +43,13 @@ class GeneratedApiTest {
         }
     }
 
+    @Test
+    fun `execute custom UseExportedLanguagesTask`() {
```

Comment:
That's not consequently applied in this project here so far, however, I am generally in favor of [expressing readable requirements with test case names](https://www.semipol.de/posts/2021/08/on-writing-useful-unit-tests/#_use_test_case_names_to_express_readable_requirements). Right now, I have not much of a clue what is actually being tested here and in case of a failure I also wouldn't exactly know which requirement is violated from the test name.

```suggestion
    fun `UseExportedLanguagesTask has access to all used MPS langauges`() {
```

That would give a lot more hints.

Reply:
Ok, that was a good suggestion.